makefile

```diff
@@ -107,6 +107,8 @@ build/src/code/fault.o: OPTIMIZATION := -O2 -g3
 build/src/code/fault_drawer.o: CFLAGS += -trapuv
 build/src/code/fault_drawer.o: OPTIMIZATION := -O2 -g3
 
+build/src/code/jpegutils.o: CC := python3 tools/asm_processor/build.py $(CC_OLD) -- $(AS) $(ASFLAGS) --
```

Comment:
This probably indicates it should be part of libultra somehow.
Anyway I think this line should be placed with the other asm_processor ones at the bottom of the makefile.